corehq/apps/receiverwrapper/views.py

```diff
@@ -64,6 +65,11 @@
 @profile_prod('commcare_receiverwapper_process_form.prof', probability=PROFILE_PROBABILITY, limit=PROFILE_LIMIT)
 def _process_form(request, domain, app_id, user_id, authenticated,
                   auth_cls=AuthContext):
+    if not submission_rate_limiter.allow_usage(domain):
+        datadog_counter('commcare.xform_submissions.rate_limited.test', tags=[
+            'domain:{}'.format(domain),
+        ])
+    submission_rate_limiter.report_usage(domain)
```

Comment:
can we ignore this block if `settings.ENTERPRISE_MODE`

Reply:
Yeah makes total sense. In addition, once this gets a little further along, I am planning on pulling the specific rates out into localsettings or otherwise make it environment-specific. There's certainly no reason other environments would want to use the same rate-limiting figures as production